keyboards/studiokestra/galatea/rev3/config.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 studiokestra (@studiokestra)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define WS2812_PIO_USE_PIO1
+
+#define RGBLED_NUM 24
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+#define RGBLIGHT_EFFECT_BREATHING
+#define RGBLIGHT_EFFECT_RAINBOW_MOOD
+#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+#define RGBLIGHT_EFFECT_SNAKE
+#define RGBLIGHT_EFFECT_KNIGHT
+#define RGBLIGHT_EFFECT_CHRISTMAS
+#define RGBLIGHT_EFFECT_STATIC_GRADIENT
+#define RGBLIGHT_EFFECT_RGB_TEST
+#define RGBLIGHT_EFFECT_ALTERNATING
+#define RGBLIGHT_MODE_TWINKLE
```

Comment:
All of this can be represented in info.json

Reply:
helpful examples include:
- reedkeebs/alish40
- ask55
- cannonkeys/serenity

Reply:
Thanks all -  it appears there's been significant progress in RP2040 support and data-driven configurations since the last time I touched QMK. I've also gone ahead and refactored the rev2 config.h file as well - all changes tested and good.